src/IcedPea.java

```diff
@@ -1,5 +1,6 @@
 public class IcedPea extends Plant {
-
+    private  Pea pea=new Pea(20);
+    private Zombie zombie;
```

Comment:
Will need to check whether adding a zombie inside the IcedPea is correct. Probably not.